distributor/app.js

```
@@ -0,0 +1,653 @@
+
+        const closeHandler = () => {
+            dialog.classList.add('hidden');
+            resolve(null);
+        };
+
+        backdrop.onclick = closeHandler;
+        if (closeBtn) closeBtn.onclick = closeHandler;
+
+        // Show dialog
+        dialog.classList.remove('hidden');
+    });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)